py/src/app/pcasts/controllers/series_subscriptions_controller.py

```diff
@@ -3,32 +3,34 @@
 class SeriesSubscriptionsController(AppDevController):
 
   def get_path(self):
-    return '/subscriptions/<series_id>/'
+    return '/subscriptions/<series_id>'
```

Comment:
we should keep this trailing `/` as a convention.  We have it in every controller's path

Reply:
i don't see it on any apiary endpoints

Reply:
Oh yea that's not necessarily up to date w/our current conventions.  This is something we should change once we push the new code.